packages/beacon-node/src/metrics/nodeJsMetrics.ts

```diff
@@ -1,6 +1,51 @@
-import {collectDefaultMetrics, Registry} from "prom-client";
+import {EventLoopUtilization, performance} from "node:perf_hooks";
+import {collectDefaultMetrics, Histogram, Registry} from "prom-client";
 import {gcStats} from "@chainsafe/prometheus-gc-stats";
 
+/**
+ * Collects event loop utilization metrics compared to the last call
+ */
+function collectEventLoopUtilization(register: Registry, prefix?: string, intervalMs: number = 5000): () => void {
+  const key = `${prefix}_` ?? "";
+
+  const metricUtilization = new Histogram({
+    name: `${key}nodejs_eventloop_utilization`,
+    help: "Histogram of Event Loop utilization between two successive calls.",
+    registers: [register],
+    buckets: [0.001, 0.01, 0.1, 0.5, 1],
+  });
+
+  const metricIdle = new Histogram({
+    name: `${key}nodejs_eventloop_idle`,
+    help: "Histogram of Event Loop idle time between two successive calls.",
+    registers: [register],
+    buckets: [1, intervalMs / 10, intervalMs / 2, intervalMs],
+  });
+
+  const metricActive = new Histogram({
+    name: `${key}nodejs_eventloop_active`,
```

Comment:
this is in seconds? sould be reflected in the metric name

```suggestion
    name: `${key}nodejs_eventloop_active_seconds`,
```